docs/fundamentals/gateway/custom-gateway.md

```diff
@@ -0,0 +1,326 @@
+(custom-gateway)=
+# Customize the Gateway
+A Jina Gateway is customizable and can be implemented in a similar way Executors are implemented.
+With Custom Gateways, Jina gives power back to the users, by allowing to implement any server, protocol and interface on
+the gateway level. This means you have more freedom in:
+* Defining and exposing your own API Gateway interface to clients. You can define your JSON schema or protos,...
+* Choosing your favourite server framework.
+* Choosing the protocol used to serve your app.
+
+Customization is allowed different components:
```

Comment:
"_Customize_ different components:"

Reply:
I want to highlight that the customization procedure can be performed through several mediums (that I'm listing afterwards

Reply:
```suggestion
Customization is allowed for different components:
```
This is grammatically correct.